modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -241,6 +241,12 @@ struct IEUnit {
             cv::util::throw_error(std::logic_error("Unsupported ParamDesc::Kind"));
         }
 
+        for (const auto& output : outputs) {
+            if (output.second->getPrecision() == IE::Precision::FP16) {
+                output.second->setPrecision(IE::Precision::FP32);
+            }
+        }
```

Comment:
Does it simply works like this? One can set this precision and IE will be forced to provide it?

Reply:
Forgot to remove it

Reply:
>>> Does it simply works like this? One can set this precision and IE will be forced to provide it?
Must work if corresponding plugin implement it, because `preprocessing` is done on `IE` core, but `postprocessing` like this is plugin responsibility so far.

Reply:
I believe we can provide API to configure input/output descriptor in future.